docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,208 @@
+SELECT
+  DISTINCT user
+FROM
+  exec
+WHERE
+  command LIKE '%/etc/passwd%';
+```
+
+![exec passwd](../../img/access-monitoring/exec_passwd.png)
+
+- Select the count of unique IP addresses associated with each user cert over different event dates:
+```sql
+SELECT
+  event_date,
+  identity_user AS user,
+  COUNT(DISTINCT cert_create.identity_client_ip) AS unique_ip_count
+FROM
+  cert_create
+GROUP BY
+  event_date,
+  identity_user
+ORDER BY
+  event_date;
+```
+
+![cert ip](../../img/access-monitoring/cert_create_ip.png)
+
+
+- Selects users who interacted with the Teleport cluster from more than one different IP address:
+```sql
+SELECT * FROM (SELECT
+  event_date,
+  identity_user AS user,
+  COUNT(DISTINCT cert_create.identity_client_ip) AS unique_ip_count
+FROM
+  cert_create
+GROUP BY
+  event_date,
+  identity_user
+ORDER BY
+  event_date) AS data WHERE unique_ip_count > 1;
+```
+
+![cert ip count](../../img/access-monitoring/cert_create_ip_count2.png)
+
+- Sow all the unique IP addresses that were used by the user 'admin-annie':
+```sql
+SELECT
+  DISTINCT identity_client_ip
+FROM
+  cert_create
+WHERE identity_user = 'admin-annie'
+```
+![ip user address](../../img/access-monitoring/ip_user_address.png)
+
+- Show access requests and their reviews:
+```sql
+SELECT
+  *
+FROM
+  access_request_create, access_request_review
+WHERE
+  access_request_create.id = access_request_review.id
+```
+
+![access request details](../../img/access-monitoring/access_request_details.png)
+
+
+- Show details about access request and review:
+```sql
+SELECT
+  request.user, request.reason, request.roles, request.resource_ids, review.reviewer, review.state
+FROM
+  access_request_create as request, access_request_review as review
+WHERE
+  request.id = review.id
+```
+![access request](../../img/access-monitoring/access_request.png)
+
+
+### Query Examples with Assist AI
+
+AI assisted query is a feature of the Teleport Access Monitoring Query editor.
+It provide SQL Query Suggestion suggestions based on the audit query schema available in your Teleport cluster.
+
+
+![assist query](../../img/access-monitoring/assist_query_1.png)
+
+
+In the result the Assist based on human text will generate the SQL query:
+![assist query result](../../img/access-monitoring/assist_query_2.png)
+
+# Access Monitoring Reports
+Access Monitoring provides pre built reports to help you monitor and analyze access patterns in your Teleport cluster.
+
+- Privileged Access Report
+    The `Privileged Access Report` reports offers valuable insight into identifying weak security events across infrastructure.
+    The report allows to identify the following weak security events:
+
+    - Database sessions with weak security
+    - SSH sessions with weak security
+    - Kubernetes API calls with weak security
+    - Privileged Postgres sessions
+    - Kube Execs
+    - Long lived certificates
+    - Long-lived join tokens
+    - Root SSH sessions
+    - System Kubernetes API calls
```

Comment:
Can we expand each of these to:

1. Give brief explanation of what it shows.
2. Show the underlying SQL query it runs.
3. Provide recommendation similar to what we show in the UI.

I would turn each of these into a sub-section as well e.g.

```
### Database sessions with weak security

... explanations, SQL, recommendation

### SSH sessions with weak security

...
```

Screenshots for each individual section aren't necessary here IMO, just a single screenshot you have below should be fine.